tuftool/tests/create_command.rs

```diff
@@ -95,8 +95,7 @@ fn create_command() {
     // Ensure the snapshot.json file is correct
     assert_eq!(repo.snapshot().signed.version.get(), snapshot_version);
     assert_eq!(repo.snapshot().signed.expires, snapshot_expiration);
-    assert_eq!(repo.snapshot().signed.meta.len(), 2);
-    assert_eq!(repo.snapshot().signed.meta["root.json"].version.get(), 1);
```

Comment:
Root metadata is no longer included in snapshot metadata. So snapshot has 1 less entry, and there is no root.json.